### ISSUE TITLE
Stars on GitHub repos pinned to profile do not update

### ISSUE THREAD

**Issue by RohitAwate** [forem/forem#212]
# _BUG_
### Current Behavior
Looks like you folks are hard coding the number of stars on a repository when it is first pinned to the profile. It doesn't update.

### Expected Behavior
Updating the value on every page request. Was this ignored to improve loading time?

### Screenshots
GitHub Repo:
![screenshot from 2018-05-03 13-16-44](https://user-images.githubusercontent.com/23148259/39565143-859eda6c-4ed4-11e8-906f-22d2426258e5.png)

Dev.to profile:
![screenshot from 2018-05-03 13-17-13](https://user-images.githubusercontent.com/23148259/39565146-899284ac-4ed4-11e8-9e11-d329f029e123.png)

**Comment by Zhao-Andy** [comment]
Hey @RohitAwate, thanks for the heads up. We'll take a look into it!

**Comment by maestromac** [comment]
Hey @RohitAwate , this is resolved. It may not be updating immediately but it will 😅. Let us know if otherwise!

**Comment by wangonya** [comment]
Hey there! Was just about about to open a similar issue when I saw this. Not sure this really works as expected. I have repos that I pinned months ago and others that I just pinned recently. None are updating the stars count. How long exactly should this take? Should I open a new issue for this? Thanks.

**Comment by maestromac** [comment]
Hello @wangonya can you verify that you are still experiencing this issue? If so, we will reopen this.

**Comment by RohitAwate** [comment]
I can confirm that the star count doesn't update unless I re-pin the repository to my profile.

**Comment by jessleenyc** [comment]
@maestromac perhaps this is fixed for new repos pinned vs. repos that have already been pinned?

**Comment by maestromac** [comment]
@jessleenyc that might've been the case.

**Comment by MSF-Jarvis** [comment]
I just unpinned and re-pinned my repos and the star count still appears to not update. How long should I wait for an update before we can safely say that the bug is still around?

**Comment by maestromac** [comment]
Hey @MSF-Jarvis can you check if it's updated now?

**Comment by MSF-Jarvis** [comment]
> Hey @MSF-Jarvis can you check if it's updated now?

Sorry, it isn't. https://github.com/MSF-Jarvis/viscerion is at 62 stars on GitHub but still at 60 stars on https://dev.to/msfjarvis which was when I pinned it.

**Comment by gautamkrishnar** [comment]
This issue is still there:
<img width="1440" alt="Screenshot 2019-09-12 at 11 41 01 AM" src="https://user-images.githubusercontent.com/8397274/64758534-72b4d900-d552-11e9-96c2-cbb08469b4ce.png">
 Shows 525 stars for my repo that has 800+ stars: https://github.com/gautamkrishnar/nothing-private

### Workaround 
The only workaround is removing the repo and re-adding it.

**Comment by VladimirMikulic** [comment]
As of May 2020, the issue is still present.

**Comment by gautamkrishnar** [comment]
One workaround is removing and readding the repo @VladimirMikulic

**Comment by VladimirMikulic** [comment]
Thanks, I knew for this workaround but I would rather avoid workarounds :)

**Comment by rhymes** [comment]
@VladimirMikulic @gautamkrishnar we're currently working on a revamp of this functionality, we'll keep you all updated. We should be able to address this issue as well.

Thank you for reminding us :-)

**Comment by gautamkrishnar** [comment]
Awesome 🎉

**Comment by cmgorton** [comment]
I just checked everyones linked repos and profiles and this appears to be fixed now. All repos and pinned repos have the same amount of stars.  Can anyone confirm this is working and not just the above mentioned work around?

**Comment by gautamkrishnar** [comment]
@cmgorton looks like it is working fine. :tada:


I can observe that the count is one less than the actual number.

https://dev.to/gautamkrishnar

![Screenshot_20210127-062642.jpg](https://user-images.githubusercontent.com/8397274/105926894-e3d22f00-6068-11eb-9430-c213833fbe1a.jpg)

![Screenshot_20210127-062922.jpg](https://user-images.githubusercontent.com/8397274/105926968-02382a80-6069-11eb-80a5-101e1179658a.jpg)

**Comment by cmgorton** [comment]
Thank you for confirming!

**Comment by MSaddamKamal** [comment]
This issue still exist the count of stars on my profile is totally mismatched by numbers of stars on my individual repositories. Please have a look at below screenshot.
![screencapture-github-MSaddamKamal-2022-09-16-02_46_35](https://user-images.githubusercontent.com/75113661/190514785-82014e14-244e-4310-b000-e974066d53f7.png)

**Comment by olegease** [comment]
maybe consider removing the stars count badge to avoid confusion, because this functionality not working for ... like 6 or 7 years?
